Makefile

```diff
@@ -31,25 +32,34 @@ all: docker
 gobuild:
 	./scripts/build false
 
+# create output directories
+out:
+	mkdir ./out
```

Comment:
I'd recommend `mkdir -p` for all of these.  Plus, you can do it on the same line:

`mkdir -p ./out/test-artifacts ./out/cni-plugins`